spec/support/devise_helper.rb

```diff
@@ -0,0 +1,14 @@
+module DeviseRequestSpecHelpers
```

Comment:
This is already available in devise https://github.com/plataformatec/devise/wiki/How-To:-sign-in-and-out-a-user-in-Request-type-specs-(specs-tagged-with-type:-:request) check first method